avocado/utils/asset.py

```diff
@@ -125,7 +125,10 @@ def _download(self, url_obj, asset_path):
             with FileLock(asset_path, 1):
                 shutil.copy(temp, asset_path)
                 self._create_hash_file(asset_path)
-                return self._verify_hash(asset_path)
+                if not self._verify_hash(asset_path):
```

Comment:
The second phrase of the commit message is missing a closure.

Reply:
Nice catch, @clebergnu. textwidith=72 and lines starting with #issue number is not a good idea. Git will mark interpret as a comment and ignore.

Fixed on my last force-push.